compiler/rustc_codegen_ssa/src/traits/backend.rs

```diff
@@ -111,6 +111,9 @@ pub trait CodegenBackend {
         codegen_results: CodegenResults,
         outputs: &OutputFilenames,
     ) -> Result<(), ErrorGuaranteed>;
+
+    /// Returns the list of intrinsics that should not use their fallback body
```

Comment:
This is the list of intrinsics *with* defaulted bodies that should *not* use them?

If so, I think a better name here would have the word `overridden` or something to convey that you don't need to put *all* intrinsics in here (e.g. `black_box`). Or at least this should be clear in the comment.